.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: "Build"
+on:
+  pull_request:
+  push:
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Can we pin down the ubuntu version here? To make this more reproducible?

Reply:
Sure, updated.